apps/portal/jaggery.conf

```diff
@@ -1,79 +1,80 @@
 {
```

Comment:
we cannot add licence headers to a json data containing file as they only allow to contain json data.